.github/workflows/contribs.yml

```diff
@@ -11,7 +14,7 @@ jobs:
       - name: Contribs
         uses: carlescufi/action-contribs@main
         with:
-          github-token: ${{ secrets.NCS_GITHUB_TOKEN }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This will not work because because this action needs to list the members of the nrfconnect org, which is not available with `GITHUB_TOKEN`

Reply:
Hmm that's a tricky one. Is this list public? What are the permissions of `NCS_GITHUB_TOKEN`?